boa/cli/transmute.py

```diff
@@ -24,9 +26,31 @@ def sizeof_fmt(num):
         return "1 byte"
 
 
-def main(args):
-    from mamba.mamba_api import transmute as mamba_transmute
+def transmute_task(f, args):
+    filename = os.path.basename(f)
+    outpath = os.path.abspath(args.output_directory)
+
+    if f.endswith(".tar.bz2"):
+        filename = filename[:-8]
+        outfile = os.path.join(outpath, filename + ".conda")
+    elif f.endswith(".conda"):
+        filename = filename[:-6]
+        outfile = os.path.join(outpath, filename + ".tar.bz2")
+    else:
+        console.print("[bold red]Transmute can only handle .tar.bz2 and .conda formats")
+
+    mamba_transmute(f, outfile, args.compression_level)
```

Comment:
can we have a single line print here, just to let the user know that files are being worked on?

Reply:
done!